### ISSUE TITLE
not an issue per say but have a look at ICU4J

### ISSUE THREAD

**Issue by coldfusionPaul** [timblair/coldfusion-inflector#3]
ICU4J has plural rules, plural currency, rule-based spellout etc., for a boatload of locales. right now, you're only doing english & you'll be constantly updating it.

**Comment by adamcameron** [comment]
Agreed.  Seems like a slight wheel-reinvention.  But having a CF wrapper for ICU4J would be handy!
